content/en/platform/corda/5.2/developing-applications/ledger/flows/_index.md

```diff
@@ -55,14 +56,18 @@ For example, Alice is a participant on the network and wishes to agree a ledger
 Now that a connection is established, Alice and Bob can communicate to agree a ledger update by passing a series of messages back and forth, as defined by the flow steps.
 
 ### Subflows
+
 Flows can be composed by starting a flow as a subprocess in the context of another flow. The flow that is started as a subprocess is known as a subflow. The parent flow waits until the subflow returns.
 
 ## Flow Library
+
 Corda provides a library of flows to handle common tasks. As a result, you do not have to redefine the logic behind common processes such as:
+
 * Notarizing and recording a transaction.
 * Gathering signatures from counterparty nodes.
 * Verifying a chain of transactions.
 Rather than exposing a library of subflows, this functionality is invoked via the `UtxoLedgerService` class. For example, calling the `finalize()` method invokes a subflow under the hood.
 
 ## Concurrency
-Virtual nodes can have multiple active flows running at once. Flows are serialized to the message bus whenever they enter a blocking state. For example, when waiting on I/O or a networking call. This allows flows to be active for long periods of time, even during interruptions such as node restarts and upgrades. Instead of waiting for the flow to become unblocked, the node immediately starts work on any other scheduled flows, returning to the original flow at a later date. A flow will suspend whenever it needs to perform an operation using platform APIs. Flows can survive a failover event, allowing them to migrate to another {{< tooltip >}}flow worker{{< /tooltip >}} process if required.
+
+Virtual nodes can have multiple active flows running simultaneously. When appropriate, Corda serializes flows to an out-of-process state manager. This allows flows to be active for long periods of time, even during interruptions such as node restarts and upgrades. Instead of waiting for the flow to become unblocked, the node immediately starts work on any other scheduled flows, returning to the original flow at a later date. A flow will suspend whenever it needs to perform an operation using platform APIs. Flows can survive a failover event, allowing them to migrate to another {{< tooltip >}}flow worker{{< /tooltip >}} process if required.
```

Comment:
> A flow will suspend whenever it needs to perform an operation using platform APIs.

The problem here is that it _**literally depends on the platform API used by the Corda Application**_... some calls might suspend the flow (asynchronous) and some won't (synchronous). Unless we include the full list in the documentation (or Javadocs, for what's worth), customers won't have any idea regarding when the suspension will happen.